### ISSUE TITLE
json-encoded throws error "Cannot JSON encode object of class" error

### ISSUE THREAD

**Issue by dmitrig01** [damballa/abracad#16]
Code:
```clj
(def s (avro/parse-schema {:name "test" :type "record" :fields [{:name "id" :type "long"}]}))
(avro/json-encoded s {:id 123456})
```

Expected behavior: it returns a JSON string :-)

Actual behavior: `JsonGenerationException Cannot JSON encode object of class: class abracad.avro$schema_QMARK_: abracad.avro$schema_QMARK_@2cafcfc9  cheshire.generate/generate (generate.clj:147)`

**Comment by dmitrig01** [comment]
Perhaps I'm using it wrong?

**Comment by llasram** [comment]
This is a bug, fixed in 460ffe4, but without subsequent release.  I've just released a new version (0.4.12) which includes the fix.

**Comment by dmitrig01** [comment]
woah awesome, thanks so much for the quick response!
